instance/models/mixins/openedx_database.py

```diff
@@ -514,6 +514,8 @@ def _get_redis_settings(self):
             "EDXAPP_CELERY_BROKER_VHOST": str(self.redis_server.instance_db),
             "EDXAPP_CELERY_BROKER_USE_SSL": self.redis_server.use_ssl_connections,
 
+            "EDXAPP_SINGLE_BEAT_IDENTIFIER": self.redis_username,
```

Comment:
@gabor-boros  Just checking -- the sandbox has all of these values set explicitly in its extra_configuration_settings, but it doesn't need to anymore, because it's got `redis provisioned`, right?

```
EDXAPP_CELERY_BROKER_HOSTNAME: "redis-ocim.net.opencraft.hosting:6379"
EDXAPP_CELERY_BROKER_TRANSPORT: redis
EDXAPP_CELERY_BROKER_USE_SSL: false
EDXAPP_CELERY_BROKER_VHOST: 0
EDXAPP_CELERY_PASSWORD: "<REDACTED>"
EDXAPP_CELERY_USER: "<REDACTED>"
```

Reply:
Do we also need these set, or is this optional per-instance?

```
EDXAPP_ENABLE_CELERY_BEAT: true
EDXAPP_CELERYBEAT_SCHEDULER: django_celery_beat.schedulers:DatabaseScheduler
```

Reply:
> but it doesn't need to anymore, because it's got redis provisioned, right?

It wouldn't be needed, it was a copy-pasted config.

> Do we also need these set, or is this optional per-instance?

These parameters are necessary per instance where we need that.